tuxemon/item/effects/evolve.py

```diff
@@ -42,10 +43,14 @@ class EvolveEffect(ItemEffect):
     monster_evolve: str
 
     def apply(self, target: Monster) -> EvolveEffectResult:
-        monster_evolve = self.monster_evolve
-
-        if any(d["monster_slug"] == monster_evolve for d in target.evolutions):
-            self.user.evolve_monster(target, monster_evolve)
+        if self.monster_evolve == "random":
+            choices = [d for d in target.evolutions if d.path == d.path.item]
```

Comment:
This doesn't sound right...

`d.path == d.path.item`

Reply:
Originally it was "item", I wanted to centralize and remove the "item", I guess I need to put again "item"

Reply:
I hope it sounds better now.